x-pack/plugins/security_solution/public/common/components/exceptions/exceptionable_windows_mac_fields.json

```diff
@@ -0,0 +1,4 @@
+[
+    "file.path.caseless",
+    "file.target_path.caseless"
+]  
```

Comment:
Do we need to include other `caseless` fields in this list, like `Target.process.executable.caseless`?

Reply:
that's a good question.

@ferullo @gabriellandau - in addition to `file.path` for Linux, should I restrict all `caseless` fields?

Reply:
spoke with @ferullo offline, I'll make distinctions like this for all of the `*.caseless` fields between Linux and Mac/Windows.

Reply:
Sounds good. Thanks!